CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [3.6.2]
+### Added
+- `WATER_MAP` jobs accept a `minimization_metric` parameter for iterative flood depth jobs. 
+
+### Changed
+- `WATER_MAP` jobs now use `None` as a default for `known_water_threshold`
```

Comment:
Should be at least a minor bump because we've added functionality, but technically I think it should be a major bump because we've changed a default parameter value, which is a breaking change: https://semver.org/